src/components/ContactInfo/ContactInfo.vue

```diff
@@ -346,7 +348,7 @@ export default class ContactInfo extends Vue {
   /** Initialize the contact info. */
   @Watch('businessContact', { deep: true, immediate: true })
   private initializeContactInfo () {
-    this.contactInfo = { ...this.businessContact }
+    this.contactInfo = { ...this.businessContact, confirmEmail: this.businessContact.email }
```

Comment:
Ah, good one.

Reply:
This saves us handling it at the parent level!